wp-content/mu-plugins/upload-mimes.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+// phpcs:disable
+/**
+ * Plugin Name: Upload Mimes
+ * Description: Adds SVGs mime type to Media uploader to enable support for SVG files.
+ * Plugin URI: https://github.com/cityofnewyork/nyco-wp-docker-boilerplate/wp/wp-content/mu-plugins/upload-mimes.php
+ * Author: NYC Opportunity
+ * Author URI: nyc.gov/opportunity
+ */
+// phpcs:enable
+
+add_filter('upload_mimes', function($mimes) {
+  $mimes['svg'] = 'image/svg+xml';
+
+  return $mimes;
+});
```

Comment:
@theasteve everything looks good except this file. I'm not sure why it's different between these branches. Can you un-track this change so the PR is scoped to versioning? Thanks!